doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -481,6 +481,12 @@ This section provides detailed lists of changes by :ref:`script <scripts>`.
 
   * SPDX License List database updated to version 3.18.
 
+* :ref:`partition_manager`:
+
+  * Added:
```

Comment:
Add or fix?

Commit message reads fix, changelog reads add.

Reply:
The configuration is new so it was added. The change in the yaml file with the partition description also fixed the issue of misaligned offset of the Settings partition for non-tfm builds.

Reply:
Also by adding the align constraint the issue was fixed